chapters/speech-to-text-api/pages/speaker-diarization.mdx

```diff
@@ -11,16 +11,36 @@ Speaker diarization is the process of detecting multiple speakers in an audio, a
 Diarization is enabled by sending the `diarization` parameter in the transcription request:
 ```json
 {
-  "audio_url": "<your audio URL>"
+  "audio_url": "<your audio URL>",
   "diarization": true
 }
 ```
 
+## Enabling diarization enhanced (bêta)
```

Comment:
```suggestion
## Enabling enhanced diarization (beta)
```
English documentation (Beta). Change it below as well.

Also `diarization enhanced` is really weird sounding. Prefer `enhanced diarization`

Reply:
This name will probably change after product review, it's a quick placeholder for some users to test it